analysis_options.yaml

```diff
@@ -1,4 +1,4 @@
-include: package:lint/analysis_options.yaml
+include: package:flutter_lints/flutter.yaml
```

Comment:
we're not reverting to a less restrictive lint package

Reply:
Sorry about that, I misread and thought that you were using the default Dart `lint`, not the third party package `lints` which is in fact more restrictive.